### PR TITLE
HomeTask_1

### PR DESCRIPTION
**Pull request by MaksimKuznets**

### CODE REVIEW

File: homeTask_1/homeTask_1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+namespace homeTask_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Console.WriteLine("Hello World!");
+            //Задача- вывести в консоль максимальные и минимальные допустимые значения для
+            //типов данных byte, short, int, long, float, double, uint, ulong.
+            byte bMax = byte.MaxValue;
```
Comment (EgorAntonovich) [inline homeTask_1/homeTask_1/Program.cs]: It's not necessary to create min/max/name variables for each data type. These values are already stored in System.
Thus, we simply fill the memory with data that already exists. Please refactor it.

File: homeTask_1/homeTask_1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace homeTask_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Console.WriteLine("Hello World!");
```
Comment (EgorAntonovich) [inline homeTask_1/homeTask_1/Program.cs]: Senseless comment

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]

File: homeTask_1/homeTask_1/Program.cs (hunk 1 above)
Comment (MaksimKuznets) [reply homeTask_1/homeTask_1/Program.cs]: Hi @EgorAntonovich. No problem. I will fix it.

File: homeTask_1/homeTask_1/Program.cs (hunk 2 above)
Comment (MaksimKuznets) [reply homeTask_1/homeTask_1/Program.cs]: Yes, I've fixed it. @EgorAntonovich would you mind reviewing this task?

Comment (EgorAntonovich) [reply homeTask_1/homeTask_1/Program.cs]: Yep. In my way it looks better.

**Review by EgorAntonovich — APPROVED** [review]
Merge your PR please.

File: homeTask_1/homeTask_1/Program.cs (hunk 2 above)
Comment (MaksimKuznets) [reply homeTask_1/homeTask_1/Program.cs]: @EgorAntonovich Can I merge this feature into the main branch?

**Pull request merged**